mlir/include/mlir/Dialect/OpenMP/OpenMPOps.td

```diff
@@ -1416,6 +1416,50 @@ def TargetOp : OpenMP_Op<"target",[IsolatedFromAbove, OutlineableOpenMPOpInterfa
   let hasVerifier = 1;
 }
 
+//===---------------------------------------------------------------------===//
+// 2.14.6 target update data Construct
```

Comment:
I see the section numbers from the spec are used across all op definitions, but I think this might cause confusion since section numbers change across versions. I see, for example, `2.12.7 Declare Target Directive` where the section number comes from 5.0, I think, while here we use `2.14.6` which comes from 5.1 AFAICT.

Maybe instead of section numbers, we should use the latest spec version (partially) supported by the current implementation of the op?

Reply:
I seem to remember from a recent call (might be misremembering) that references to section numbers in the spec are supposed to refer by default to the 4.5 version, and that section numbers from other versions have to specify which one. I'm not sure that this is always the case, though. Can you confirm @kiranktp?

Reply:
> Everyone agreed on this approach...
That I did not know. Thanks for the clarification.